www/js/util.js

```diff
@@ -1625,6 +1659,12 @@ function addChatMessage(data) {
     }
 
     pingMessage(isHighlight);
+    
+    //Strip HTML before passing it to the notification
```

Comment:
This belongs in `showMessage()` since the need for it is local to that.  Also allows you to optimize and not create an unnecessary DOM element for the conversion unless notifications are enabled and this message would result in one.